### PR TITLE
Refactoring, Style changes and add Package Uploading

### PR DESCRIPTION
**Pull request by exception**

### CODE REVIEW

**Comment by redxdev** [comment]
I'm fine with most of these changes (pending my ability to test them, which I'll do shortly) but I'd rather you remove the "maintained by" line. If you'd like credit in the repository, I'll add a contributors file.

**Comment by redxdev** [comment]
There is now a CONTRIBUTORS.md file, add yourself to it and remove your change from the top of the readme.

File: README.md [hunk 1]
```diff
@@ -1,15 +1,16 @@
 # PlayPen Visual Interface (PVI)
 
-PVI is a JavaFX-based client for the PlayPen server management framework.
+PVI is a JavaFX-based client for the PlayPen server management framework. Developed by [The Chunk](https://github.com/PlayPen/), maintained by [@exception](https://github.com/exception/).
```
Comment (redxdev) [inline README.md]: Remove this line's change, add yourself to CONTRIBUTORS.md.

**Review by redxdev — CHANGES_REQUESTED** [review]
Fix the readme, then you should be good.

**Comment by exception** [comment]
Done, @redxdev, CONTRIBUTORS.md update is on https://github.com/PlayPen/PVI/pull/2

**Review by redxdev — APPROVED** [review]

**Pull request merged**